Detectors/TRD/base/macros/PrintRun2DigitsAndRun3

```diff
@@ -0,0 +1,112 @@
+#if !defined(__CINT__) || defined(__MAKECINT__)
+#include <TClonesArray.h>
```

Comment:
Might it make sense to put these Run2 macros under a special folder?
Maybe it would still make sense to put the ".C" suffix and add these macros to the list of
non-checked macros.

Reply:
Probably both suggestions make sense.

Reply:
That entire directory is aliroot dependent, will make it clear in a readme.